src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1226,6 +1230,55 @@ public void canSearchItemsByLocation() throws Exception {
       is(mainLibraryItem.getString("id")));
   }
 
+  @Test
+  public void itemHasLastCheckInPropertiesWhenTheyAreSet() throws Exception {
```

Comment:
Can we set here a more specific exception? Not common 'Exception'? (And in the next test)

Reply:
Sure.